app/mixins/event-wizard.js

```diff
@@ -219,22 +219,22 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
         const valid = preSaveActions.call(this);
         if (valid) {
           this.set('data.event.state', 'draft');
-          this.sendAction('save');
+          this.save();
```

Comment:
sendAction is not working at some places but closures work well.
eg - event creation failed at step 1
Please refer the error https://github.com/fossasia/open-event-frontend/pull/6203#pullrequestreview-560599208

Reply:
So should I revert changes for step 1?

Reply:
Yes, test all changes and revert whatever is not working